api/v1beta2/cloudstackmachine_types.go

```diff
@@ -63,7 +63,7 @@ type CloudStackMachineSpec struct {
 
 	// Optional affinitygroupids for deployVirtualMachine
 	// +optional
-	AffinityGroupIDs []string `json:"affinitygroupids,omitempty"`
+	AffinityGroupIDs []string `json:"affinityGroupIDs,omitempty"`
```

Comment:
I don't think we should change this now. EKS-A is already and other customers may also be using this field and changing the serialization/deserialization patterns feels risky, with no gain. I'm inclined to keep these as they are and fix later on in an intentional way

Unless this is strictly for the v1beta2 types, and not v1beta1? Same for the other changes involving json

Reply:
Indeed, this is only on v1beta2 types.

That's why I wanted to jamb this into this release. It won't be easy to do outside of an API version bump.